Dockerfile

```diff
@@ -20,14 +20,12 @@ RUN apt-get update \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN cd /root \
-    && wget https://downloads.wkhtmltopdf.org/0.12/0.12.5/wkhtmltox_0.12.5-1.stretch_amd64.deb --no-verbose \
-    && dpkg -i wkhtmltox_0.12.5-1.stretch_amd64.deb
+    && wget https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.stretch_amd64.deb --no-verbose \
+    && dpkg -i wkhtmltox_0.12.6-1.stretch_amd64.deb
 
 # Enables continously calling a command and piping the output to STDOUT, viewable via docker logs
 RUN printf '#!/bin/bash\nwhile sleep 1; do\n    "$@"\ndone' >> /usr/bin/watch-docker \
     && chmod +x /usr/bin/watch-docker
 
 COPY --from=composer /app/vendor /app/vendor
 COPY . /app
-
-RUN ln -s /app/bin/md2resume /usr/bin/md2resume

```

Comment:
Are you sure this is no longer needed?

Reply:
No, that shouldn't have been removed, thought it was a line of code I had added for testing, my bad!